app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,19 @@
+module SessionsHelper
+  # Logs in the given user.
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id])
```

Comment:
toán tử `||=` có tác dụng là gì vậy em?

Reply:
toán tử này là kiểm tra xem nếu @current_user chưa có thì sẽ thực hiện User.find_by(id: session[:user_id]) còn @current_user đã có thì @current_user = @current_user
giống như là @current_user là nil hoặc false thì nó thực hiện lệnh bên phải ạ
em hiểu như vậy có đúng không ạ